example/e2e/detoxTest.spec.js

```diff
@@ -88,6 +88,36 @@ describe('Example', () => {
     }
   });
 
+  async function userOpensPickerTz({mode, display, interval, type}) {
```

Comment:
again, if you can give this a little nicer name, that would be appreciated. We want the code to be accessible to anyone, so that contributing is easier, thank you!